recipes/r-garnett/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '0.0.1' %}
```

Comment:
You should maybe use the actual release binary from https://github.com/cole-trapnell-lab/garnett/releases, and set this to that (0.1.4).